src/multi_vector_simulator/F1_plotting.py

```diff
@@ -855,52 +856,70 @@ def create_plotly_flow_fig(
         figure object
     """
 
-    fig = go.Figure()
-    styling_dict = get_fig_style_dict()
-    styling_dict["gridwidth"] = 1.0
+    def create_plot(
```

Comment:
Why do you add a new function definition without docstrings, eventhough the function still basically is `create_plotly_flow_fig()`?

Reply:
We need to run through this plotting functionality (now called `create_plot()`) twice, in case SOC is part of `assets_list`. The flow is the following:

1) first preparation of `assets_list`
```
    assets_list = list(df_plots_data.columns)
    assets_list.remove("timestamp")
```
2) Check if SOC of a storage is in `assets_list`...
```
    if any(SOC in item for item in assets_list):
        # plot SOC separately
        adapted_title = plot_title.replace("power", "storage SOC")
```
...and create SOC plot...
```
        create_plot(
            flow_list=[s for s in assets_list if SOC in s],
            filename=f"SOC_{file_name}",
            y_legend="SOC in %",
            plot_title=adapted_title,
        )
```
...and remove SOC from `assets_list` to create flow plot (step 3) without SOC).
```
        # remove SOC as it is provided in % and does not fit to the flows plot
        assets_list = [s for s in assets_list if SOC not in s]
```
3) Call inner func `create_plot()` (again) to create flows plot:
```
    # create flows plot (without SOC)
    fig = create_plot(flow_list=assets_list)

    return fig
```